### PR TITLE
fix(main): Closing file before removing

### LINKED ISSUE #3440: Log file error "Critical: Unable to move logs"
OS: Windows 7
qTox version: (version numbers in menu `Settings → About`)  
Commit hash:  8d225a8c97a36838092b325ca830c14546f3a0c6 and newer
toxcore:  
Qt:  5.4.2
Hardware:  
…
https://gist.github.com/towlie/9419eea500e99bb95a4a9804d89dd880
Logging itself still works.

### PR DESCRIPTION
**Pull request by Diadlo**
Fix #3440

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -167,9 +167,13 @@ int main(int argc, char *argv[])
     if (QFileInfo(logfile).size() > 1000000)
     {
         qDebug() << "Log file over 1MB, rotating...";
-		
+
+        // close old logfile (need for windows)
+        if(mainLogFilePtr)
```
Comment (zetok) [inline src/main.cpp]: space between `if` and `(` ?

Comment (Diadlo) [inline src/main.cpp]: It's just copy-paste. Should I replace all `if(` in this function?

Comment (zetok) [inline src/main.cpp]: You could :)

Comment (Diadlo) [inline src/main.cpp]: Fixed

**Pull request merged**